src/main/java/uk/gov/ida/matchingserviceadapter/MatchingServiceAdapterModule.java

```diff
@@ -304,8 +305,8 @@ public String getHubConnectorEntityId(MatchingServiceAdapterConfiguration config
     @Provides
     @Singleton
     @Named("AcceptableHubConnectorEntityIds")
-    public String[] getAcceptableHubConnectorEntityIds(MatchingServiceAdapterConfiguration configuration) {
-        return configuration.isEidasEnabled() ? configuration.getEuropeanIdentity().getAcceptableHubConnectorEntityIds() : new String[0];
+    public List<String> getAcceptableHubConnectorEntityIds(MatchingServiceAdapterConfiguration configuration) {
+        return configuration.isEidasEnabled() ? configuration.getEuropeanIdentity().getAcceptableHubConnectorEntityIds(configuration.getMetadataEnvironment()) : new LinkedList<>();
```

Comment:
This could just be a new empty`ArrayList` instead of a `LinkedList` (which has more specialisation). Sorry for the pedantry 😧.